src/clj/runbld/notifications/slack.clj

```diff
@@ -8,15 +8,18 @@
             [runbld.notifications :as n]
             [runbld.store :as store]
             [stencil.core :as mustache]
-            [elasticsearch.document :as doc]))
+            [elasticsearch.document :as doc]
+            [robert.bruce :refer [try-try-again] :as try]))
 
 (defn api-send
   "Make the Slack REST API call"
   [opts js hook]
   (do
     ((opts :logger) "NOTIFYING SLACK")
-    (http/post hook
-               {:body js})))
+    (try-try-again
+     {:sleep 250 :tries 10 :decay try/exponential}
```

Comment:
Fixed in https://github.com/elastic/runbld/pull/53/commits/e1d1fdcc3c437843aaeebd884ffeef39e280ac84.